squad/frontend/templates/squad/knownissues.jinja2

```diff
@@ -0,0 +1,139 @@
+{% extends "squad/base.jinja2" %}
+
+{% block content %}
+
+<h2>{{ _('Known Issues') }}</h2>
+
+{% with items=results %}
+{% include "squad/_pagination.jinja2" %}
+{% endwith %}
+
+<div>
+    <div class='row row-bordered'>
+        <div class='col-md-12 col-sm-12 filter'>
+          <a id="searchLink"><button type='button' class='btn btn-primary fa fa-search'></button></a>
+          <input name='search' id='search' type='text' placeholder='{{ _('Filter results ...') }}' value='{{search}}' />
+        </div>
+    </div> 
+
+    <table class='test-results'>
+        <thead>
+            <th>{{ _('Known Issue') }}</th>
+            {% for env in results.environments %}
+            <th>{{env}}</th>
+            {% endfor %}
+        </thead>
+        {% for test, issues in results.results.items() %}
+        {% set test_history_url = url('legacy_test_history', args=[project.group.slug, project.slug, test]) %}
+        <tr id='test-{{test|slugify}}' ng-show="match('test-{{test|slugify}}')">
+            <td><a href="{{ test_history_url }}">{{test}}</a></td>
+            {% for env in results.environments %}
+                {% with result=issues[env] %}
+                    <td>
+                        {% if result %}
+                                <a href='#' data-toggle="modal" data-target="#info_modal" data-status="{{ result }}"><span data-toggle="tooltip" data-placement="right" title="{{ _('Show info') }}" class='fa fa-info-circle'></span></a>
+
+                        {% else %}
+                                {{ _('n/a') }}
+                        {% endif %} 
+                    </td>
+                {% endwith %}
+            {% endfor %}
+        </tr>
+        {% endfor %}
+    </table>
+
+</div>
+
+<!-- Modal HTML -->
+<div id="info_modal" class="modal fade" role="dialog">
+  <div class="modal-dialog" role="document">
+    <div class="modal-content">
+      <div class="modal-header">
+        <button type="button" class="close" data-dismiss="modal" aria-hidden="true">&times;</button>
+        <h4 class="modal-title">{{ _('Known-issue Details') }}</h4>
+      </div>
+      <div class="modal-body">
+        <div id="url" style="display: none;">
+          <p><strong>{{ _('Url:') }}</strong></p>
+          <a id="url_inner" href=""></a>
+        </div>
+        <div id="notes" style="display: none;">
+          <p><strong>{{ _('Notes:') }}</strong></p>
+          <div id="notes_inner">
+            <pre></pre>
+          </div>
+        </div>
+        <div id="active" style="display: none;">
+          <p><strong>{{ _('Active:') }}</strong></p>
+          <div id="active_inner">
```

Comment:
It would look better if you displayed FA tick or x instead of 'true' or 'false' string in the modal dialog.

Reply:
@mwasilew this is not fixed.